config/logs.js

```diff
@@ -66,21 +66,40 @@ module.exports = {
       }
 
       // Sanitize OAuth client secrets
+      //
+      // This will display "00000...00000" in ELK.
       if (typeof metadata.request.query.client_secret === 'string') {
-        // display first/last three characters but scrub the rest
-        const sanitizedSecret = `${metadata.request.query.client_secret.slice(0, 3)}...${metadata.request.query.client_secret.slice(-3)}`;
+        // display first/last five characters but scrub the rest
+        const sanitizedSecret = `${metadata.request.query.client_secret.slice(0, 5)}...${metadata.request.query.client_secret.slice(-5)}`;
         metadata.request.query.client_secret = sanitizedSecret;
       }
 
+      // Sanitize OAuth client secrets found in Headers
+      //
+      // This will display "Basic 00000...00000" in ELK.
+      if (
+        typeof metadata.request.headers.authorization === 'string'
+        && metadata.request.headers.authorization.indexOf('Basic') !== -1
+      ) {
+        const sanitizedSecret = `${metadata.request.headers.authorization.slice(0, 11)}...${metadata.request.headers.authorization.slice(-5)}`;
+        metadata.request.headers.authorization = sanitizedSecret;
+      }
+
       // Sanitize JWTs, which can allow anyone to masquerade as this user.
       // We do this by removing the signature at the end, which only our
       // server can generate. That way, we can decode and inspect the payload
       // for debugging purposes, without risking the use of the JWT by devs
       // who have access to ELK.
-      if (typeof metadata.request.headers.authorization === 'string') {
+      if (
+        typeof metadata.request.headers.authorization === 'string'
+        && metadata.request.headers.authorization.indexOf('Bearer') !== -1
```

Comment:
this line was the fix: checking for `Bearer` inside the contents of the auth Header